src/main/java/pl/wedding/management/model/employee/Salary.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.model.employee;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Builder
+class Salary {
+
+    private final BigDecimal finalSalary;
```

Comment:
Jeżeli mamy stawkę godzinową to nie powinniśmy ustawiać finalSalary. To będziemy wyliczali w jakimś serwisie.